clients/python/setup.py

```diff
@@ -3,6 +3,7 @@
 setup(
     name="scale-llm-engine",
     python_requires=">=3.7",
-    version="0.0.0.beta31",
+    version="0.0.0.beta32",
```

Comment:
also pyproject.toml and llmengine/__init__.py

Reply:
done